10up/autoshare-for-twitter
# Replace custom build process with WP-Scripts

### Description of the Change
PR replaces current custom build process with WP-Scripts also removes some legacy files which no longer needed

Closes #193 

### How to test the Change
1. Run `npm install`
1. Run `npm run build`
1. Make sure build files are generated successfully.
1. Smoke test plugin and make sure everything working fine

### Changelog Entry
> Changed - Replace custom build process with WP-Scripts

### Credits

Props @iamdharmesh @peterwilsoncc 


### Checklist:
